com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -117,17 +120,36 @@ internal static void DontSaveToLayout(EditorWindow wnd)
 
         void OnEnable()
         {
-            m_CoreConvertersList = new List<RenderPipelineConverter>();
-
             // This is the drop down choices.
             m_ConverterContainers = TypeCache.GetTypesDerivedFrom<RenderPipelineConverterContainer>();
+            foreach (var cont in m_ConverterContainers)
+            {
+                var container = (RenderPipelineConverterContainer)Activator.CreateInstance(cont);
+                m_Containers.Add(container);
+                m_ContainerChoices.Add(container.name);
+            }
+
+            if (m_ConverterContainers.Any())
+            {
+                GetConverters();
+            }
+        }
+
+        void GetConverters()
+        {
+            m_CoreConvertersList.Clear();
+            m_ConverterStates.Clear();
+            m_ItemsToConvert.Clear();
```

Comment:
Should these not also get cleared if the` m_ConverterContainers.Any(`) check fails?

Reply:
Done